src/internal.c

```diff
@@ -3328,8 +3328,11 @@ static int DoKexDhReply(WOLFSSH* ssh, byte* buf, word32 len, word32* idx)
 #ifdef WOLFSSH_SMALL_STACK
     if (sigKeyBlock_ptr)
         WFREE(sigKeyBlock_ptr, ssh->ctx->heap, DYNTYPE_PRIVKEY);
+    #ifndef WOLFSSH_NO_ECDSA
+        if (key_ptr)
```

Comment:
can you please adjust the spacing here, the indent causes it to look like it is in the if(sigKeyBlock_ptr) block.